manifests/input.yaml

```
@@ -86,6 +86,11 @@ work/INPUT/access-om3-025deg-rof-remap-weights.nc:
   hashes:
     binhash: 72b02f07b141ec1471c7bd3d41498863
     md5: 41657fc054785cc0e1741dd3639c3c16
+work/INPUT/bottom_roughness.nc:
+  fullpath: /g/data/vk83/prerelease/configurations/inputs/access-om3/mom/tidal_external_files/global.25km/2025.05.15/bottom_roughness.nc
+  hashes:
+    binhash: f58e3a3fe5c58aa9d9c60f81796fb07e
+    md5: 38c7e37b9ba2418cdef82e7265a7c885
```

These hashes have changed. In the future, rather than modify input files in place, please just create a new version

```suggestion
    binhash: 2107d37df1e8695f44b552645440aa1f
    md5: 061dec8e4022c0dadccd10db8d31c567
```